index.js

```diff
@@ -68,7 +68,44 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector('#header')
+header.addEventListener("click", function(event) {
+  debugger
+  toggleColor(event.target)
+})
+
+
 /***** Deliverable 2 *****/
+const likes = document.querySelector('.likes')
+const button = document.querySelector('.like-button')
+
+function addlikes() {
+  let totalLikes = parseInt(likes.textContent)
+  totalLikes++
+  likes.textContent = `${totalLikes} Likes` 
+}
+
+button.addEventListener("click", addlikes)
```

Comment:
Looks good! The key here is that you have to _read_ some information from the DOM to get the current number of likes, increment it, and then _write_ it back to the DOM. Nice work!